MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -333,23 +351,33 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkups
 
   vtkSmartPointer< vtkPolyData > cleanedPolyData = vtkSmartPointer< vtkPolyData >::New();
```

Comment:
No need for creating a smart pointer for this, just use a simple pointer. Until cleanPointPolyData filter is not deleted, the pointer will remain valid.